### PR TITLE
taxonomy: some ingredients for CIQUAL

### PR DESCRIPTION
**Pull request by teolemon**
### What
- some ingredients for CIQUAL

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -77880,6 +77905,13 @@ ciqual_food_code:en:36024
 ciqual_food_name:en:Chicken, leg, meat, raw
 ciqual_food_name:fr:Poulet, cuisse, viande, cru
 
+
+en:Chicken drumstick
+fr:pilon de poulet
+ciqual_food_code:en:36022
+ciqual_food_name:en:Chicken, drumstick, raw
+ciqual_food_name:fr:Poulet, pilon, cru
```
Comment (teolemon) [inline taxonomies/ingredients.txt:77913]: It seems a different entity than what we have

**Review by stephanegigandet — APPROVED** [review]

**Comment by stephanegigandet** [comment]
Looks good, thanks!

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (teolemon) [inline taxonomies/ingredients.txt:77913]:
```suggestion
# en:Chicken drumstick
# fr:pilon de poulet
# ciqual_food_code:en:36022
# ciqual_food_name:en:Chicken, drumstick, raw
# ciqual_food_name:fr:Poulet, pilon, cru
```

Comment (teolemon) [inline taxonomies/ingredients.txt:77913]:
```suggestion
# en:Chicken drumstick
# fr:pilon de poulet
# ciqual_food_code:en:36022
# ciqual_food_name:en:Chicken, drumstick, raw
# ciqual_food_name:fr:Poulet, pilon, cru
```

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -77880,6 +77905,13 @@ ciqual_food_code:en:36024
...
+
```
Comment (teolemon) [inline taxonomies/ingredients.txt:77914]:
```suggestion

```

**Comment by teolemon** [comment]
thanks @alexgarel @stephanegigandet

**Pull request merged**